src/qdrant_client/builders/query.rs

```diff
@@ -14,6 +15,16 @@ impl QueryPointsBuilder {
     }
 }
 
+impl QueryPointGroupsBuilder {
+    pub fn add_prefetch(mut self, prefetch_query: impl Into<PrefetchQuery>) -> Self {
+        match self.prefetch {
+            Some(ref mut prefetch) => prefetch.push(prefetch_query.into()),
+            None => self.prefetch = Some(vec![prefetch_query.into()]),
+        }
+        self
+    }
```

Comment:
Though this is fine, we could simplify to something like [this](https://doc.rust-lang.org/std/option/enum.Option.html#method.get_or_insert_with) (untested):

```suggestion
    pub fn add_prefetch(mut self, prefetch_query: impl Into<PrefetchQuery>) -> Self {
        self.prefetch.get_or_insert_with(Vec::new).push(prefetch_query.into());
        self
    }
```

Reply:
Thanks, applied it to the second builder as well https://github.com/qdrant/rust-client/pull/181/commits/25277d6f16319435abb5f553f23a4834f66f3b69